### DISCUSSION TITLE
Como faço para colocar o retrato do personagem no site?

### DISCUSSION THREAD

**Discussion by MeetLime291621** [alyssapiresfernandescefet/openrpg#9] [Q&A] [answered]
Eu não estou conseguindo botar o retrato do personagem no site. Pensei que era pra botar o link da imagem (hospedada em um site), mas mesmo assim não funciona.
![image](https://user-images.githubusercontent.com/107946712/175108856-0917cedd-573f-471d-9522-10db381d1258.png)

**Comment by lucaspiresfernandes** [comment, marked as answer]
Olá! Então, para usar o avatar você precisa inserir exatamente o link da imagem. O link que você botou é o link do imgur, que por sua vez mostra a imagem. Então, pra resolver esse problema você faz o seguinte:

1 - Vá até a sua imagem, clique com o botão direito nela e clique em "Copiar o endereço de imagem".

![image](https://user-images.githubusercontent.com/71353674/175118437-2d974930-342e-4f09-8707-62ade77c0b2f.png)

2 - Com esse endereço de imagem, cole no respectivo avatar.

Nesse caso, o link dessa imagem seria o seguinte:

https://i.imgur.com/ed4c7mJ.png
